editor/create_dialog.cpp

```diff
@@ -50,18 +50,14 @@ void CreateDialog::popup_create(bool p_dont_clear, bool p_replace_mode, const St
 	}
 
 	if (p_replace_mode) {
-		search_box->set_text(p_select_type);
+		search_box->set_text(p_current_type);
 	}
 
 	search_box->grab_focus();
 	_update_search();
 
 	if (p_replace_mode) {
-		if (!p_select_name.is_empty()) {
-			set_title(vformat(TTR("Convert %s from %s"), p_select_name, p_select_type));
-		} else {
-			set_title(vformat(TTR("Convert %s"), p_select_type));
-		}
+		set_title(vformat(TTR("Change type of \"%s\""), p_current_name));
```

Comment:
We use Title Case for titles, so this  should be:
```suggestion
		set_title(vformat(TTR("Change Type of \"%s\""), p_current_name));
```